### ISSUE TITLE
laravel 5.3 This dependency was not found in node_modules 

### ISSUE THREAD

**Issue by Melcus** [JeffreyWay/laravel-mix#182]
Installed mix as per installation guide. `webpack.mix.js` file contains
```
let mix = require('laravel-mix');

mix.js('resources/assets/js/frontend/app.js', 'public/test_js')
 .version();
```
In `package.json` -> 
```
 "dev": "cross-env NODE_ENV=development webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
```
When I run `npm run dev` I get
```
This dependency was not found in node_modules:                         
                                                                       
* c:\laragon\www\xxxxxx\resources\assets\js\frontend\app.js   
```
The file is there, it's not missing, why is it looking in node_modules? 

Using windows 10 and Laragon.

**Comment by JeffreyWay** [comment]
It looks like node_modules if it can't find it locally. It thinks your referencing an installed module.

Hmm- not sure. You're 100% positive there's no typo or missing directory path? I'm not sure why it would tell you that, if the file does indeed exist.

**Comment by Melcus** [comment]
![Image](http://i.imgur.com/TxCj2y5.png)

Yeah, it's there.

**Comment by QWp6t** [comment]
you got any weird isolation stuff going on? maybe a permissions issue? maybe a symlink that isn't being followed properly?

**Comment by Melcus** [comment]
Everything standard, nothing altered

**Comment by JeffreyWay** [comment]
Yeah something weird is going on. Nuke your node_modules directory, and do a `npm install` from scratch.

**Comment by Melcus** [comment]
Did it with -save and without, tried with yarn and npm... nothing seems to
help

**Comment by JeffreyWay** [comment]
Can you try creating a fresh app from scratch, and then trying to compile. We'll see if it's unique to your current app at all. 

```
laravel new test
npm install
npm run dev
```

**Comment by QWp6t** [comment]
@Melcus I just noticed that you said you're using Laragon.

I'm not familiar with Laragon or how it works under the hood, but when I visited the site just now, one of the first things it says is...

> Laragon has an **isolated environment** ...

which is what I initially assumed would be causing this issue:
> you got any weird isolation stuff going on? maybe a permissions issue? maybe a symlink that isn't being followed properly?

Anway, I'll stay out of this now. Just wanted to point out that Laragon could be the issue here.
